torchvision/extension.py

```diff
@@ -23,10 +27,25 @@ def _register_extensions():
 try:
     _register_extensions()
     _HAS_OPS = True
+    def _has_ops():
+        return True
 except (ImportError, OSError):
     pass
 
 
+def _assert_has_ops():
```

Comment:
I should do something similar for the video and image ops, but I'll do that in a follow-up PR once we define more precisely if this implementation is enough